[devtools] WNDT 109 updates + translation template (#4897)
GoogleChrome/developer.chrome.com: site/en/blog/new-in-devtools-109/index.md

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/55382b27eff3539c8aba42ea501eb8de4f7ba57c #}

## [Experimental] Enhanced UX in managing breakpoints {: #breakpoint-redesign }

{% Aside %}
To enable the experiment, check **Enable re-designed Breakpoint Sidebar Pane in the Sources Panel** under **Settings** > **Experiments**.
{% endAside %}

The current **Breakpoints** pane provides little visual aid in overseeing all breakpoints. On top of that, frequently used actions are hidden behind the context menu.

This experimental UX redesign aims at bringing structure into the **Breakpoints** pane and allow developers to have quick access to commonly used features, in particular editing and removing breakpoints.

These are some highlights:

- Both pause options are in the **Breakpoints** pane and labeled with text to make it more self-explanatory.

> - Breakpoints are grouped by file, ordered by line/column number, and collapsible.**

Reviewer: ```suggestion
- Breakpoints are grouped by file, ordered by line or column number. You can collapse and expand them.**
```

> - New options to remove and edit breakpoint when hovering over a breakpoint or file name in the **Breakpoint** pane.

Reviewer: ```suggestion
- New options to remove and edit a breakpoint when hovering over the breakpoint or a file name in the **Breakpoint** pane.
```

Read the full changes in our [RFC (closed)](https://github.com/ChromeDevTools/rfcs/discussions/3) and leave your feedback [here](https://crbug.com/1394686).

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ytfyl8qK5rkHQRTS3sXf.png", alt="Show Breakpoint pane before and after the redesign.", width="800", height="684" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f2140378e0bb1687b263c226de01b741487ff324 #}
Chromium issues: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 

## [Experimental] Automatic in-place pretty print {: #pretty-print }

{% Aside %}
To enable the experiment, check **Automatically pretty print in the Sources panel** under **Settings** > **Experiments**.
{% endAside %}

> The **Sources** panel now automatically pretty print minified source files in-place. You can click on the **pretty print button `{ }` to underdo it.

Reviewer: ```suggestion
The **Sources** panel now automatically pretty-prints minified source files in place. You can click the **pretty print** button `{ }` to undo it.
```

> Previously, the **Sources** panel shows minified content by default. Developers need to click on the pretty print button manually to format the content. On top of that, the pretty printed content is not displayed in the same file, but in another `::formatted` tab.

Reviewer: ```suggestion
Previously, the **Sources** panel showed minified content by default. You had to click the pretty print button manually to format the content. On top of that, the pretty-printed content wasn't displayed in the same file, but in another `::formatted` tab.
```
